oeps/architectural-decisions/oep-0014-proc-archive-repos.rst

```diff
@@ -65,27 +68,30 @@ First, if the repository is public, and a part of Open edX releases, follow thes
 
 2. If someone does wish to take ownership of the repository, email the internal edX developers mailing list to see if there are any objections. If there are no objections after 2 work days, post to `Open edX Deprecation Announcements`_ that the transfer will take place.
 
-3. Create an IT help ticket for the repository to be transferred to the new organization.
+**Code Review: How could we make the above internal-to-edX step something public? I think publicizing on the deprecation announcement forum and on Slack (I suggest #general and #depr-slash-n-burn) should be enough; organizations are welcome to do more, but each org's processes shouldn't be reflected in an OEP.**
+
+3. Create a new `tCRIL engineering task`_ for the repository to be transferred to the new organization.
```

Comment:
@kdmccormick I think this is right, what do you think?

Reply:
@sarina Maybe `GitHub Request on the tCRIL board` (that way gets auto-assigned and a `github-request` label) instead of a general tCRIL task, but otherwise looks solid.